src/main/resources/conf/global-default.xml

```diff
@@ -0,0 +1,200 @@
+    <value>../bin/dist_pytrain.sh</value>
+  </property>
+
+    <property>
+    <description>Training data path on hdfs</description>
+    <name>shifu.application.training-data-path</name>
+    <!-- <value>/user/pengzhang/ModelSets/cam2015-BNN/tmp/NormalizedData/</value> -->
+    <value>/user/webai/tmp/data/</value>
+  </property>
+
+  <!-- Task configurations -->
+  <property>
+    <description>JVM opts for each TaskExecutor.</description>
+    <name>shifu.task.executor.jvm.opts</name>
+    <value>-Xmx1536m</value>
+  </property>
+
+  <property>
+    <description>Frequency, in milliseconds, for which TaskExecutors should heartbeat with AM.</description>
+    <name>shifu.task.heartbeat-interval</name>
+    <value>1000</value>
+  </property>
+
+  <property>
+    <description>How many missed heartbeats before declaring a TaskExecutor dead.</description>
+    <name>shifu.task.max-missed-heartbeats</name>
+    <value>25</value>
+  </property>
+
+  <!-- AM configurations -->
+  <property>
+    <description>How many times a failed AM should retry.</description>
+    <name>shifu.am.retry-count</name>
+    <value>1</value>
+  </property>
+
+  <property>
+    <description>AM memory size, requested as a string (e.g. '2g' or '2048m').</description>
+    <name>shifu.am.memory</name>
+    <value>2g</value>
+  </property>
+
+  <property>
+    <description>Number of AM vcores to use.</description>
+    <name>shifu.am.vcores</name>
+    <value>1</value>
+  </property>
+
+  <!-- PS configurations -->
+  <property>
+    <description>Parameter server memory size, requested as a string (e.g. '2g' or '2048m').</description>
+    <name>shifu.ps.memory</name>
+    <value>4g</value>
+  </property>
+
+  <property>
+    <description>Number of vcores per parameter server.</description>
+    <name>shifu.ps.vcores</name>
+    <value>2</value>
+  </property>
+
+  <property>
+    <description>Number of parameter servers to request.</description>
+    <name>shifu.ps.instances</name>
+    <value>2</value>
+  </property>
+<!-- 
+  <property>
+    <description>Number of back-up parameter servers to request.</description>
+    <name>shifu.ps.instances.backup</name>
+    <value>2</value>
+  </property>
+-->
+  <!-- Worker configurations -->
+  <property>
+    <description>Timeout, in milliseconds for the user's python processes before forcibly killing them.</description>
+    <name>shifu.worker.timeout</name>
+    <value>0</value>
+  </property>
+
+  <property>
+    <description>Worker memory size, requested as a string (e.g. '2g' or '2048m').</description>
+    <name>shifu.worker.memory</name>
+    <value>10g</value>
+  </property>
+
+  <property>
+    <description>Number of vcores per worker.</description>
+    <name>shifu.worker.vcores</name>
+    <value>2</value>
+  </property>
+
+  <property>
+    <description>Number of workers to request.</description>
+    <name>shifu.worker.instances</name>
+    <value>1</value>
+  </property>
```

Comment:
should be set according to # of splits

Reply:
I intend to leave this to user to config.
If we do it automatically, beside number of workers, we also could set workers' memory and cores depends on data size and model complicity. I think we could add this as a issue in issue list